hubbot

```diff
@@ -415,6 +431,9 @@ def expire_work_dirs(github_visible_commits):
             if not dry_run:
```

Comment:
It's always one of the elements returned by list_work_dirs(). Should we double check that list_work_dirs() work correctly?